### ISSUE TITLE
Error: Could not read file /MyComponent.vue

### ISSUE THREAD

**Issue by pierosavi** [CyCraft/vue-intellisense#6]
Hello,
when using the library with a single Vue component it works wonderfully, when passing a directory it always gives this error.

I tried with the most basic setup, easily reproducible: 

inside `src/components/` I have only MyComponent.vue that is:

```
<template>
	<div>
	</div>
</template>

<script lang="ts">
import Vue from 'vue';

export default Vue.extend({
});
</script>

<style scoped>
</style>
```

`vue-int --input ./src/components/MyComponent.vue --output ./lib/vetur` works fine

`vue-int --input ./src/components/ --output ./lib/vetur --recursive` (Or without recursive) gives error

```
Error: Could not read file /MyComponent.vue
    at Object.<anonymous> (/Users/******/*******/vue-ts-sfc-starter/node_modules/vue-docgen-api/dist/parse.js:98:27)
```

**Comment by pierosavi** [comment]
vue: v2.6.10
vue-intellisense: v0.1.1
node: v14.15.3

**Comment by pierosavi** [comment]
Looking through the code I found where the issue comes from:

https://github.com/CyCraft/vue-intellisense/blob/ae3e09879f24b7331e51c6618906a00f1ab7e856/packages/scripts/src/listFiles.ts#L37

resolvePaths being default to false resolves MyComponent.vue relative to *fullpath*/src/components - so just /MyComponent.vue, but that doesn't in my folder root, where the script is executed

**Comment by mesqueeb** [comment]
@pierosavi
your fix commit is live now in `v0.1.2`.
let me know if you are still having issues!
